hcl2template/types.packer_config.go

```diff
@@ -401,6 +405,15 @@ func (cfg *PackerConfig) getCoreBuildPostProcessors(source SourceUseBlock, block
 			if moreDiags.HasErrors() {
 				continue
 			}
+
+			if cfg.bucket != nil && cfg.bucket.Validate() == nil {
```

Comment:
Do you mind explaining this one to me?

Reply:
This means that for each post-processor the metadata will be pushed right after, right?